crates/matrix-sdk-crypto/src/olm/group_sessions/sender_data.rs

```diff
@@ -215,6 +215,7 @@ enum SenderDataReader {
         legacy_session: bool,
     },
 
+    #[serde(alias = "SenderUnverifiedButPreviouslyVerified")]
     VerificationViolation(KnownSenderData),
```

Comment:
There are a whole bunch of other renames in #4067 that we need to check for; notably `OwnUserIdentityVerifiedState` (https://github.com/matrix-org/matrix-rust-sdk/pull/4067/files#diff-d28a9285dd9fd0d1ced29b1986dc92d7a754bb846a24cab48ec78073a1581166R874) which was the one that caused problems for Valere, but also `VerificationLevel`, and there may be others.

Reply:
Sorry, was rushing to find a fix and missed the other places. Will expand this.

Reply:
Added commits dealing with the other cases